index.html

```diff
@@ -2,12 +2,16 @@
 
 <html>
   <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
     <!-- Stylesheet -->
     <link rel="stylesheet" href="styles/styles.css" />
     <title>BookDB - Home</title>
   </head>
 
   <body id="main-container">
+    <!-- Main App JS Script -->
+    <script src="index.js"></script>
```

Comment:
why you put script here ?
please fully understand why we put script in **body** vs in **head** and what does **defer** and **async**